src/Main.java

```diff
@@ -1,57 +1,58 @@
+import java.time.LocalDate;
+
 public class Main {
-    public static boolean counterAge( int year) {
-        if (year % 4 != 0) {
-            return false;
-        } else if (year % 100 != 0) {
-            return true;
-        } else {
-            return year % 400 == 0;
+    public static void findALeapYear(int year) {
```

Comment:
метод печатает, поэтому лучше назвать так
```suggestion
    public static void printALeapYearOrNot(int year) {
```